src/lib/moment/momentUtils.ts

```diff
@@ -33,6 +45,18 @@ export const getCurrentPeriod = () => {
   return year + "-" + month.toString().padStart(2, "0");
 }
 
+export const getNextPeriod = () => {
+  let year = moment().year();
```

Comment:
This can probably just be something like `moment().add(1, 'month')...` instead of this type of logic.